code/modules/clothing/suits/marine_armor.dm

```diff
@@ -67,6 +70,15 @@
 		armor_overlays["lamp"] = null
 	user?.update_inv_wear_suit()
 
+/obj/item/clothing/suit/storage/marine/apply_custom(image/standing)
+	if(length(armor_overlays))
+		var/image/I
+		for(var/i in armor_overlays)
+			I = armor_overlays[i]
```

Comment:
Is this length really needed

Reply:
no idea, ctrl c ctrl v, seeing how this is CM code  i woulnd't dare change that one

Reply:
the answer is no